plugin.program.autowidget/resources/lib/add.py

```diff
@@ -267,17 +267,30 @@ def _copy_path(path_def):
     if not group_id:
         return
 
+    progress = xbmcgui.DialogProgressBG()
+    progress.create("AutoWidget", utils.get_string(30143))
+    progress.update(1, "AutoWidget", utils.get_string(30144))
+
     group_def = manage.get_group_by_id(group_id)
-    files, hash = refresh.get_files_list(path_def["file"]["file"])
+    files, hash = refresh.get_files_list(path_def["file"]["file"], background=False)

```

Comment:
This ensures that exploded/cloned groups don't get slapped with a "Loading content..." item and call it a day :+1:

Reply:
ah so thats happening in the background anyway right? seems reasonable.